test/cql-pytest/test_sstable.py

```diff
@@ -36,3 +37,21 @@ def test_twcs_optimal_query_path(cql, test_keyspace, scylla_only):
         # in the debug build.
         nodetool.flush(cql, table)
         assert 1 == len(list(cql.execute(f"SELECT * FROM {table} WHERE pk = 0 BYPASS CACHE")))
+
+# Reproduces https://github.com/scylladb/scylladb/issues/17625.
+# Without the fix, it trips ASAN.
+def test_big_key_index_reader(cql, test_keyspace, scylla_only):
```

Comment:
nitpick (not important now, the PR was already merged) - this test doesn't need to be scylla_only, it should work just as well on Cassandra.

Reply:
I take my comment back - you use BYPASS CACHE which is Scylla-only.